src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -473,14 +471,14 @@ protected function _bundleTranslatedFields($entity)
         }
 
         $results = $this->_findExistingTranslations($find);
-        $model = $this->config('conditions.model');
+        $alias = $this->_table->alias();
 
```

Comment:
If this is just the table alias, `$this->config('model')` will be useless, it needs to be the model set in the config, and if no model is set it needs to then be the table alias. The way you've changed it means if I've added this behavior, from say `PostsTable` and set the `model` in the config to `Articles`, this wouldn't find any existing translations since in the DB the `model` is `Articles`.